cmd/api/handlers-queries.go

```diff
@@ -0,0 +1,174 @@
+package main
+
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/jmpsec/osctrl/pkg/queries"
+	"github.com/jmpsec/osctrl/pkg/settings"
+	"github.com/jmpsec/osctrl/pkg/utils"
+)
+
+// GET Handler to return a single query in JSON
+func apiQueryShowHandler(w http.ResponseWriter, r *http.Request) {
+	incMetric(metricAPIReq)
+	utils.DebugHTTPDump(r, settingsmgr.DebugHTTP(settings.ServiceAPI), false)
+	vars := mux.Vars(r)
+	// Extract name
+	name, ok := vars["name"]
+	if !ok {
+		incMetric(metricAPIErr)
+		apiErrorResponse(w, "error getting name", nil)
+		return
+	}
+	// Get query by name
+	query, err := queriesmgr.Get(name)
+	if err != nil {
+		incMetric(metricAPIErr)
+		if err.Error() == "record not found" {
+			log.Printf("query not found: %s", name)
+			apiHTTPResponse(w, JSONApplicationUTF8, http.StatusNotFound, ApiErrorResponse{Error: "query not found"})
+		} else {
+			apiErrorResponse(w, "error getting query", err)
```

Comment:
Can we guarantee this err doesn't contain any sensitive data? It looks like it would be returned to the client (does the client get all that info then)?